tests/rptest/tests/e2e_shadow_indexing_test.py

```diff
@@ -141,6 +141,10 @@ def setUp(self):
     def test_create_or_delete_topics_while_busy(self):
         self.logger.info(f"Environment: {os.environ}")
         if os.environ.get('BUILD_TYPE', None) == 'debug':
+            # Trigger node allocation to avoid test failure on asking for
+            # more nodes than it used.
+            _ = self.preallocated_nodes
```

Comment:
nit: if the parent has some method explicitly named like `preallocate_nodes` which can be called here this might be easier to understand.